src/classes/BDI_DataImportService.cls

```diff
@@ -363,10 +377,11 @@ global with sharing class BDI_DataImportService {
     * @return String either the String value or an empty String ''
     */
     public String strNull(String str) {
```

Comment:
Two questions:
- Does this need to be public?
- What would you think of a method name of `replaceNullWithEmptyString(String str)`?

Reply:
Naming is hard, thus chiming in with more suggestions :) Perhaps `defaultToEmptyIfNull()`, or just `defaultToEmpty()`?

Reply:
Nice! Either of those is much better than what I came up with.

Reply:
This is used in another class, so I'm not planning to update it now. If I open up BDI_ContactService that will be another can of worms.